[2024-10-18T13:22:37.173+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-18T13:22:37.205+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.this_is_1st_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2024-10-18T13:22:37.220+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.this_is_1st_task scheduled__2022-03-04T00:00:00+00:00 [queued]>
[2024-10-18T13:22:37.224+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-18T13:22:37.299+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): this_is_1st_task> on 2022-03-04 00:00:00+00:00
[2024-10-18T13:22:37.325+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2846) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-18T13:22:37.324+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'this_is_1st_task', 'scheduled__2022-03-04T00:00:00+00:00', '--job-id', '1973', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpde8d6pr9']
[2024-10-18T13:22:37.335+0000] {standard_task_runner.py:105} INFO - Job 1973: Subtask this_is_1st_task
[2024-10-18T13:22:37.332+0000] {standard_task_runner.py:72} INFO - Started process 2868 to run task
[2024-10-18T13:22:37.443+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v3.this_is_1st_task scheduled__2022-03-04T00:00:00+00:00 [running]> on host ad4831ffc3d7
[2024-10-18T13:22:37.562+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sourabh' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='this_is_1st_task' AIRFLOW_CTX_EXECUTION_DATE='2022-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-04T00:00:00+00:00'
[2024-10-18T13:22:37.562+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-18T13:22:37.590+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-18T13:22:37.594+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world, this is my first task']
[2024-10-18T13:22:37.600+0000] {subprocess.py:86} INFO - Output:
[2024-10-18T13:22:37.601+0000] {subprocess.py:93} INFO - Hello world, this is my first task
[2024-10-18T13:22:37.602+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-18T13:22:37.638+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-18T13:22:37.641+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=this_is_1st_task, run_id=scheduled__2022-03-04T00:00:00+00:00, execution_date=20220304T000000, start_date=20241018T132237, end_date=20241018T132237
[2024-10-18T13:22:37.703+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-18T13:22:37.721+0000] {taskinstance.py:3900} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-10-18T13:22:37.723+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
